### ISSUE TITLE
improve PDF generation using buffers rather than local temp files

### ISSUE THREAD

**Issue by filipslavik** [dideher/phaistos#34]
printing/exporting leaves to PDF should not store the temp PDF file locally just to send them to the requesting client. Something along the following lines should be used : 

```python
import io

def download_pdf(request, song_id):
    buffer = io.BytesIO()
    
    # ...

    chordsheet_html.write_pdf(buffer, stylesheets=[chordsheet_css])
    buffer.seek(0)
    return FileResponse(buffer, as_attachment=True, filename=song.title + "_" + song.artist + ".pdf")
```

(example taken from here: https://stackoverflow.com/questions/68385270/using-weasyprint-to-create-file-response)

**Comment by filipslavik** [comment]
@gstam you can use the same approach as with #37

**Comment by filipslavik** [comment]
Fixed in #39
